app/assets/scripts/map.js

```diff
@@ -97,37 +130,62 @@ function create_marker(resource){
   async.each(values,
     function(value, callback){
       markerToAdd.addListener('click', function() {
+        $("#map").show();
+        $("#resource-info").hide();
+
         var contentDiv = document.createElement('div');
-        var strong = document.createElement('strong');
+        var resName = document.createElement('strong');
+        $(resName).html(resource.name);
         var br = document.createElement('br');
-        $(strong).html(resource.name);
-        $(contentDiv).append(strong, br, resource.address);
+
+        // information pane to replace map
+        // TODO: refactor how the page elements are generated
```

Comment:
Yeah with my TODO refactor I'm going to create a separate task for this generating of info pages refactor since I think there'll be a lot there in and of itself. I'm currently looking into Mustache, Handlebars etc. as potentially useful but still have to do more research. If there isn't anything particularly useful then I'm hoping to have the corresponding Jinja file contain most of div components and just have this function populate the text content as an alternative instead of having both text and lot of div elements.